wire-ios/Wire-iOS/Generated/Strings+Generated.swift

```diff
@@ -4521,7 +4521,10 @@ internal enum L10n {
         }
         internal enum RestoreBackupFailed {
           /// Your history could not be restored.
-          internal static let message = L10n.tr("Localizable", "registration.no_history.restore_backup_failed.message", fallback: "Your history could not be restored.")
+          ///  Error: %@
+          internal static func message(_ p1: Any) -> String {
+            return L10n.tr("Localizable", "registration.no_history.restore_backup_failed.message", String(describing: p1), fallback: "Your history could not be restored.\n Error: %@")
```

Comment:
I wouldn't add the work error, just %@ to make it easier to keep other localisations

Reply:
in that case we can keep the translations as before and just append to the next line the error, right?

Reply:
yes